enlighten/LaserControlFeature.py

```diff
@@ -193,6 +185,10 @@ def disconnect(self):
     def refresh_laser_button(self):
         spec = self.multispec.current_spectrometer()
         enabled = spec.settings.state.laser_enabled if spec else False
+        if enabled:
+            self.button_toggle.setText("Laser Off")
+        else:
+            self.button_toggle.setText("Laser On")
```

Comment:
Actually, I really would prefer "Turn Laser On" and "Turn Laser Off".  When people read "Laser On" they don't necessarily know if that's "current status" or "what the button will do."  Note that some users (even some staff) are colorblind, so the CSS doesn't automatically help.